### DISCUSSION TITLE
Why should we create a lightweight and user-friendly email template editor using Grapesjs?

### DISCUSSION THREAD

**Discussion by jacksonkasi1** [GrapesJS/grapesjs#5215] [General]
I believe GrapesJS is an amazing open-source tool/product. However, I feel that the user experience might be a bit challenging. I'm not sure how many people enjoy creating websites using this tool, but I think people would appreciate making email templates with a lighter version of this tool.

**Comment by artf** [comment]
Hi @jacksonkasi1 you should keep in mind that GrapesJS is a framework for creating your builder/editor and not end user application, so it's up to you to create what a user-friendly interface would look like.
Here you can see some preset examples for email builders: [Newsletter preset](https://github.com/grapesjs/preset-newsletter), [MJML preset](https://github.com/GrapesJS/mjml)

**Reply by jacksonkasi1** [reply]
Hi @artf, thank you for your response and for pointing out that GrapesJS is a framework for creating builders and editors. I appreciate the clarification. I've checked out the Newsletter preset and MJML preset examples that you've shared, and they seem to be great starting points for creating a user-friendly email template builder. I'll explore these presets further and see how I can customize them to create an even more intuitive email template creation experience for users. If you have any other suggestions or resources, please feel free to share. Thanks again!
